src/lib/select/select.spec.ts

```diff
@@ -56,7 +64,7 @@ const LETTER_KEY_DEBOUNCE_INTERVAL = 200;
 const platform = new Platform();
 
 
-describe('MatSelect', () => {
+fdescribe('MatSelect', () => {
```

Comment:
It looks like there's a ban on `fdescribe` in tslint.json

Reply:
Is there a way that I can make this change and resubmit? Or is it necessary to wait for @crisbeto to make the change?

Reply:
I haven't addressed it yet because I'm waiting for some more feedback, @jrood.

Reply:
Ok thanks.